code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -1,4 +1,5 @@
 #define EMPOWERED_THRALL_LIMIT 5
+var/in_jaunt = 0
 
 /obj/effect/proc_holder/spell/proc/shadowling_check(var/mob/living/carbon/human/H)
 	if(!H || !istype(H)) return
```

Comment:
Add this 

 /obj/effect/proc_holder/spell/proc/shadowling_check(var/mob/living/carbon/human/H)
 	if(H.incorporeal_move == 1)
 		usr << "<span class='warning'>You cannot do this while in another dimension!</span>"
 		return